pkg/deviceshifu/deviceshifubase/deviceshifubase.go

```diff
@@ -67,6 +72,8 @@ const (
 	PowerShellStubTimeoutTolerationStr    string = "stub_toleration"
 	PythonHandlersModuleName                     = "customized_handlers"
 	PythonScriptDir                              = "pythoncustomizedhandlers"
+	SQLSettingSecret                             = "telemetry_service_sql_pwd"
+	HTTPSettingSecret                            = "telemetry_service_http_pwd"
```

Comment:
These two lines are two fixed name fields in the `Secret` with customizable name. These two fields are optional, if present they will overwrite the original password. They don't need to be present at the same time, if only one of the protocols is used.

Since a protocol has at most one endpoint, the second problem mentioned in my PR is also fine.